src/lib.rs

```diff
@@ -219,14 +232,24 @@ pub fn eject() {
 pub trait ImguiRenderLoop {
     /// Called once at the first occurrence of the hook. Implement this to
     /// initialize your data.
-    fn initialize<'a>(&'a mut self, _ctx: &mut Context, _loader: TextureLoader<'a>) {}
-
-    /// Called every frame. Use the provided `ui` object to build your UI.
-    fn render(&mut self, ui: &mut Ui);
+    fn initialize<'a>(
```

Comment:
Could you please adapt the comment here stating that `ctx` is the imgui context and `render_context` is meant to access hudhook renderers' extensions such as texture management?